Applications/Games/The Sims 4/Origin/script.js

```diff
@@ -0,0 +1,42 @@
+include("engines.wine.quick_script.origin_script");
+include("engines.wine.verbs.vcrun6sp6");
+include("engines.wine.verbs.mfc42");
+include("engines.wine.verbs.dotnet20sp2");
+include("engines.wine.verbs.vcrun2010");
+include("engines.wine.verbs.vcrun2013");
+include("engines.wine.verbs.d9vk");
+include("utils.functions.net.download");
+include("utils.functions.filesystem.files");
+
+var installerImplementation = {
+    run: function () {
+        new OriginScript()
+            .name("The Sims 4")
+            .editor("Electronic Arts")
+            .applicationHomepage("https://www.ea.com/games/the-sims/the-sims-4/pc")
+            .author("ZemoScripter")
+            .category("Games")
+            .wineVersion("4.0")
```

Comment:
Why is it only 4.0 ? AppDB says platinum with wine 4.3-staging for example.

Reply:
I've tried 4.3-staging and it's the same crash. I then tried other wine version and each time I changed it in phoenicis.cfg I've changed it in script.js as well.

Reply:
Weird.

Reply:
Every tutorial I see is about Origin not working. Not the game itself.

Reply:
Tried an older AppDB entry stating they used DXVK but still the same crash.